deps/osxunwind.def

```diff
@@ -0,0 +1,2 @@
+-nr:___register_frame:___jl_register_frame
+-nr:___deregister_frame:___jl_deregister_frame
```

Comment:
it's fine to mix and do both, although i guess that can cause confusion by the linker on which version of `__register_frame` you intended to link against (unless you just dlsym both libraries at runtime to grab this function pointer)